capstone/src/main/java/com/google/sps/servlets/CallPerspectiveServlet.java

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import java.io.BufferedReader;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.sps.data.PerspectiveInput;
+import org.json.simple.JSONObject;    
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/** Servlet that returns Perspective scoring. */
+@WebServlet("/call-perspective")
+public class CallPerspectiveServlet extends HttpServlet {
+  static final String url = "https://commentanalyzer.googleapis.com/v1alpha1/comments:analyze?key=API_KEY";
+  static final MediaType JSON = MediaType.get("application/json; charset=utf-8");
+  static final ArrayList<String> attributes = new ArrayList<String>(Arrays.asList("TOXICITY", "PROFANITY", "THREAT", "INSULT", "IDENTITY_ATTACK", "SEVERE_TOXICITY"));
```

Comment:
nit: Constants should be CAPITAL_CASE

https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
Fixed